src/cml.js

```diff
@@ -344,6 +344,7 @@ class CML {
       }
     }
     await exec('git fetch --all');
+    await exec('git config --global --add safe.directory "$PWD"');
```

Comment:
```suggestion
    await exec('git config --global --add safe.directory "$(git rev-parse --show-toplevel)"');
```

Reply:
:-1: 😅 

1. `git rev-parse` fails like any other `git` command.
2. If it didn't, what you propose would defeat the new Git security feature.

Reply:
ahh yes, fail like the git command in the line before this one? :thinking:

Reply:
`git clone git@github.com:iterative/cml.git && docker run --rm -it -v $PWD/cml:/home/runner ghcr.io/iterative/cml:latest git fetch --all`

Reply:
Yes, 👎🏼 🏆 for me. 🙈

Reply:
we both did not actually test what we wrote :see_no_evil: :test_tube:

Reply:
https://github.com/iterative/cml/pull/974/commits/63653797eeda1068b3e74f91d7aa18c90f06c249